### PR TITLE
[dbnode] Add metric for series blocks read per request

### PR DESCRIPTION
**Pull request by nbroyles**
**What this PR does / why we need it**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing and/or backwards incompatible change?**:

```release-note

```

**Does this PR require updating code package or user-facing documentation?**:

```documentation-note

```

### CODE REVIEW

File: src/dbnode/network/server/tchannelthrift/node/service.go [hunk 1]
```diff
@@ -929,10 +931,12 @@ func newFetchTaggedResultsIter(opts fetchTaggedResultsIterOpts) FetchTaggedResul
 		// for each block as opposed to acquiring in bulk).
 		opts.blocksPerBatch = 1
 	}
+	scope := opts.iOpts.MetricsScope().SubScope("fetch-tagged-iter")
```
Comment (rallen090) [inline src/dbnode/network/server/tchannelthrift/node/service.go:934]: Should we put this on `service` so we don't have to create one for each iter?

File: src/dbnode/network/server/tchannelthrift/node/service.go [hunk 2]
```diff
@@ -1075,6 +1080,8 @@ func (i *fetchTaggedResultsIter) Close(err error) {
 	i.totalMetrics.ByRange.Record(queryRange, totalFetchTime)
 	i.totalMetrics.ByDocs.Record(i.totalDocsCount, totalFetchTime)
 
+	i.totalBlocks.Update(float64(i.totalBlocksCount))
```
Comment (rallen090) [inline src/dbnode/network/server/tchannelthrift/node/service.go:1083]: Is a gauge what we want here? Won't we miss a lot of the potential fluctuation based on just what the val happens to be on each scrape?

Comment (nbroyles) [reply src/dbnode/network/server/tchannelthrift/node/service.go:1083]: Ugh, I forgot about the scrape interval. What's the standard way of getting metrics per request?

**Comment by nbroyles** [comment]
Closing this while I fix up

**Pull request closed without merging**